src/util.rs

```diff
@@ -1,9 +1,15 @@
 pub mod binary;
 
+use crate::Result;
+pub use skiplist::FixedLengthSuffixComparator as Comparator;
 pub use skiplist::{FixedLengthSuffixComparator, KeyComparator};
 use std::{cmp, ptr};
 
-pub static COMPARATOR: FixedLengthSuffixComparator = FixedLengthSuffixComparator::new(8);
+pub static COMPARATOR: FixedLengthSuffixComparator = make_comparator();
```

Comment:
What's the difference between `Comparator::new(8)`?

Reply:
No difference, just to make the interface unified. What about reverting this change?